src/app/(private)/(routes)/admin/classroom/(routes)/video/[videoId]/components/new-classroom-video-form/upload-image-module/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Icons } from '@/components/icons'
+import { toast } from '@/components/ui/use-toast'
+import { useDropzone } from 'react-dropzone'
+
+type UploadImageModuleProps = {
+  onSetPreviewFiles: (file: File[]) => void
+}
+export const UploadFilesModule = ({
+  onSetPreviewFiles
+}: UploadImageModuleProps) => {
+  const { getRootProps, getInputProps } = useDropzone({
+    maxFiles: 10,
+    accept: {
+      'application/pdf': ['.pdf'],
+      'application/msword': ['.doc', '.docx'],
+      'application/vnd.ms-excel': ['.xls', '.xlsx'],
+      'application/vnd.openxmlformats-officedocument.presentationml.presentation':
+        ['.ppt', '.pptx'],
+      'text/plain': ['.txt'],
+      'application/zip': ['.zip']
+    },
```

Comment:
I suggest defining this value as a constant so it can be reused throughout the codebase.